src/views/Shift.vue

```diff
@@ -90,10 +108,16 @@ export default {
   },
   data () {
     return {
+      ORDER_REFRESH_WAIT: 1000,
+      ORDER_REFRESH_RATE: 2000,
+      ORDER_REFRESH_TIMEOUT: 30000,
+      SHIFT_REFRESH_RATE: 30000,
       shift: null,
       order: null,
       shiftProgress: 0,
-      fetchingTimedOut: false
+      fetchingTimedOut: false,
+      updateOrderToServer: null,
+      orderBeingUpdated: false,
```

Comment:
This value is never used

Reply:
It is, in src/components/OrderCard.vue:90